src/currency.js

```diff
@@ -146,6 +146,21 @@ currency.prototype = {
     return currency(intValue /= parse(number, _settings, false), _settings);
   },
 
+  /**
+   * Compares value to another currency instance's value
```

Comment:
How should this handle amounts of different currencies? (e.g. comparing dollars to euros?)